### PR TITLE
testing  helpers, questions and create-component

### PR DESCRIPTION
**Pull request by stevenpeniche**
With this PR we can close issue #8 regarding adding unit tests.

### CODE REVIEW

File: bin/helpers.js [hunk 1]
```diff
@@ -3,8 +3,8 @@
 const Configstore = require('configstore');
 const conf = new Configstore('napp-config');
 
-const changeAllSettings = (bool) => {
-    conf.set({
+const changeAllSettings = (store=conf, bool) => {
```
Comment (tdfranklin) [inline bin/helpers.js:6]: Just curious, but why did you change the name to store here?  Just to be more verbose or was there a specific reason?

**Review by tdfranklin — APPROVED** [review]
Overall, I'm really impressed with the work you've done.  Everything looks great!  I'm just curious about the one change, but not enough to hold up approving this.  Great job and I look forward to seeing what you do next!

File: bin/helpers.js (hunk 1 above)
Comment (stevenpeniche) [reply bin/helpers.js:6]: I added that `store` parameter so that I could pass in a separate `test-napp-config` store for testing purposes. It helps avoid manipulating our local `nap-config` stores.

Comment (tdfranklin) [reply bin/helpers.js:6]: Oh, I gotcha!  Thanks for letting me know!
